corehq/apps/analytics/static/analytix/js/appcues.js

```diff
@@ -29,21 +29,19 @@ hqDefine('analytix/js/appcues', [
     $(function () {
         window.AppcuesSettings = {
             skipAMD: true,
+            enableURLDetection: true,
         };
         var apiId = _get('apiId'),
             scriptUrl = "//fast.appcues.com/" + apiId + '.js';
 
+        const allUserProperties = getIdentityProperties(
+            initialAnalytics.getNamespacedProperties('appcues'));
+        const username = allUserProperties['username'];
+        const userProperties = _.omit(allUserProperties, 'username');
+
         _logger = logging.getLoggerForApi('Appcues');
         _ready = utils.initApi(_ready, apiId, scriptUrl, _logger, function () {
-            identify(_get("username"), {
```

Comment:
i know it feels cleaner to have these new helper methods...however, for someone who is trying to understand where certain properties are being used it makes it a bit harder to grep the code and immediately understand where these properties are being generated/used. Curious on your thoughts about this? Just feels like it's on the verge of over-obfuscation.

Although, now that we've decided appcues is not something we are using for this purpose, perhaps it's also better to take out the appcues changes? Especially since the other analytics code is not using `getIdentityProperties`